### PR TITLE
Re-adding gravity_affected_blocks.json and adding "GRAVITY_AFFECTED_BLOC…

### PR DESCRIPTION
**Pull request by TerslenK**
…KS" into SlimefunTag.java

## Description
i made this pull request before, but i made some issues so this pull request should fix them

## Proposed changes
added src/main/resources/tags/gravity_affected_blocks.json
added "GRAVITY_AFFECTED_BLOCKS" into src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/SlimefunTag.java

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Comment by TerslenK** [comment]
im sure i made everything correct rn

**Comment by WalshyDev** [comment]
Why did you open a new PR rather than just committing to the old one?

**Comment by TerslenK** [comment]
> You just... Commit more

eeh

sorry i guess ¯\_(ツ)_/¯

**Comment by TerslenK** [comment]
> Just a small typo, everything else seems fine to me

sorry i dont have a good grammar

**Comment by WalshyDev** [comment]
Is there not a tag for concrete powder?

also I believe most of the tags have unit tests. If so, we should have one here too

**Review by WalshyDev — DISMISSED** [review]

File: src/main/resources/tags/gravity_affected_blocks.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+	"values" : [
+		"minecraft:dragon_egg",
+		"minecraft:anvil",
+		"minecraft:chipped_anvil",
+		"minecraft:damaged_anvil",
```
Comment (J3fftw1) [inline src/main/resources/tags/gravity_affected_blocks.json:6]:
```suggestion
		#minecraft:anvil,
```

**Review by J3fftw1 — CHANGES_REQUESTED** [review]

File: src/main/resources/tags/gravity_affected_blocks.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "values" : [
+        "minecraft:dragon_egg",
+        "minecraft:gravel",
+        "minecraft:sand",
+        "minecraft:red_sand",
+        "minecraft:scaffolding",
+        "#slimefun:concrete_powders",
+        "#minecraft:anvil",
```
Comment (J3fftw1) [inline src/main/resources/tags/gravity_affected_blocks.json:9]: move anvil above slimefun tag

Comment (TerslenK) [reply src/main/resources/tags/gravity_affected_blocks.json:9]: okay

Comment (TerslenK) [reply src/main/resources/tags/gravity_affected_blocks.json]: is it right now?

**Review by variananora — APPROVED** [review]
LGTM

**Review by svr333 — APPROVED** [review]
LGTM

**Review by TheBusyBiscuit — APPROVED** [review]
LGTM

**Review by Sfiguz7 — APPROVED** [review]

**Review by TheBusyBiscuit — APPROVED** [review]
Why hasn't this been merged yet?

**Comment by svr333** [comment]
wasnt merged because no one can but walshy and he didnt check it

**Comment by TerslenK** [comment]
finally 😅

**Pull request merged**